### ISSUE TITLE
Missing diacritics after new note

### ISSUE THREAD

**Issue by ludenticus** [pbek/QOwnNotes#755]
#### Expected behaviour                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             

Typing diacritics produces diacritics

#### Actual behaviour                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               

Diacritics are missing after creating a new note _if_ I enable `The shortcut to create a new note asks for a headline`. Restarting QON restores diacritics

#### Steps to reproduce                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             

1. Enable `The shortcut to create a new note asks for a headline`
2. Create a new note, either by `Ctrl + N` or _via_ `Note` → `New Note`
3. Try typing `á`, `à`, `ã` etc. and it will only produce `a`

#### Relevant info
Version: `17.10.7`
Platform: `linux`
Operating System: `Antergos Linux`

Let me know if you can reproduce it

**Comment by pbek** [comment]
I just tried to create a new note `á test2` under KDE Neon and it worked like it should. But I pasted the text...

**Comment by pbek** [comment]
Actually typing the text in the dialog worked too... Do you have the same problem in other line-edits in QOwnNotes or in other Qt applications?

**Comment by ludenticus** [comment]
It's weird. I disabled and re–enabled the option and now it's working as intended.

I had previously experienced this strange behavior but I ignored it. I decided to test it today after creating a new note and noticing it wasn't producing diacritics: this is when I found out about the hypothetical relationship between the option and the diacritics. After disabling/restarting/re–enabling, however, I'm no longer able to reproduce it. Maybe it had to do with ibus and Gnome. We'd better close the issue until further info.
